File: lecture_002(Normal)/l_003_sumofdigits.java
import java.util.Scanner;
public class l_003_sumofdigits {
    public static Scanner scn = new Scanner(System.in);
    public static void main(String[] args) {
        System.out.println("Enter a number:");
        int num = scn.nextInt();
        int sum=0;
        while (num != 0) {
            sum= sum + num%10;
            num/=10;
        }
        System.out.println("Sum of digits:" + sum);
    }
}

File: lecture_002(Normal)/l_003_sumofdigits.cpp
#include<iostream>
using namespace std;
int main(int args, char**argvs) {
    int num, sum=0;
    cout<<"Enter a number:";
    cin>>num;
    while(num != 0) {
        sum= sum + num % 10;
        num /= 10;
        
    }
    cout<<"Sum of digits:";
    cout<<sum;
    
    return 0;
}